apiary-metastore-listener/src/main/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListener.java

```diff
@@ -45,12 +45,16 @@
   private static final Logger log = LoggerFactory.getLogger(ApiarySnsListener.class);
 
   private static final String TOPIC_ARN = System.getenv("SNS_ARN");
+  final static String PROTOCOL_VERSION = "1.0";
+
+  private final String protocolVersion = PROTOCOL_VERSION;
   private AmazonSNSClient snsClient;
 
   public ApiarySnsListener(Configuration config) {
     super(config);
     // create a new SNS client and set endpoint
     snsClient = new AmazonSNSClient();
+    // TODO: we should probably set a default region rather than require this
```

Comment:
Replace this with AmazonSNSClientBuilder.defaultClient();

and it uses the default {{DefaultAwsRegionProviderChain}} which allows SystemEnv overrides same system env actually "AWS_REGION". It will fall back to default (I assume instance profile region) which probably will suffice 99% of the time.
You can remove the setRegion call in that case.

Reply:
Sounds like a good idea, and more standard than defining our own environment variable for region choice. More info at https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-region-selection.html. If @rpoluri agrees I will make this change (possibly via a separate ticket and PR though).

Reply:
sounds good to me. I think aws libraries use AWS_DEFAULT_REGION , not AWS_REGION.
I can make corresponding change in docker container.

Reply:
@massdosage link you added says AWS_REGION, so we should be fine. If it doesn't work I can look at changes in docker.

Reply:
OK, great, I have created a separate ticket for that here https://github.com/ExpediaInc/apiary-extensions/issues/4 so can lets consider this out of scope for this PR.